### ISSUE TITLE
cellfinder v1.3.0 blog post

### ISSUE THREAD

**Issue by adamltyson** [brainglobe/brainglobe.github.io#177]
The next version of cellfinder is likely to be a (relatively) large one, and we should write a blog post to outline the differences. This will include:
- Switch to torch backend (https://github.com/brainglobe/cellfinder/pull/396)
- Support turning off classification or detection in GUI (https://github.com/brainglobe/cellfinder/pull/402)
- Support for single z-stack tif file for input (https://github.com/brainglobe/cellfinder/pull/397)
- Cell detection optimisation (#398) by (https://github.com/brainglobe/cellfinder/pull/407)
- Pretrained models moving to `~.brainglobe/cellfinder`, how the old models can be deleted, and how to re-install them in the new location (#176) (https://github.com/brainglobe/cellfinder/pull/414)
- Maybe cell detection in torch.

cc @alessandrofelder

**Comment by IgorTatarnikov** [comment]
Here are the graphs and correlation/coefficients comparing tensorflow to torch on both **Brain 1** and **Brain 2** from the paper:

![comparison_left](https://github.com/brainglobe/brainglobe.github.io/assets/61896994/1df72272-1d51-456f-a3af-02d5e95f278c)

![comparison_right](https://github.com/brainglobe/brainglobe.github.io/assets/61896994/06443ae9-ad53-4e0c-88fe-3a6a0dcb25e6)

The timing information is below:

|  | Brain 1 - paper | Brain 1 - torch | Brain 2 - paper | Brain 2 - torch | Brain 2 - torch + new detection |
|--|--------|--------|--------|--------|--------|
| Candidate detection | 50 | 46.5 | 48 | 64.5   | 48 |
| Classification            | 33 | 35.5 | 43 | 39      | 39.5 |
| Total                         | 83 | 82     | 91 | 103.5 | 87.5 |

**Comment by adamltyson** [comment]
To clarify some points @IgorTatarnikov:
- Based on the plots, torch is (technically, very slightly) better than TF?
- The detection timings, is the main section of the plot the timings from the latest release, and "new detection" is from main (after @matham's PR)?

**Comment by IgorTatarnikov** [comment]
Correct, seems torch is a tiny bit better than TF.

For detection timings, all were run on the original machine from the paper. The "paper" columns are numbers taken from the cellfinder paper, the "torch" columns are the means of two runs using the detection code that is in the current release but with a torch backend. The last column is combining the new detection from https://github.com/brainglobe/cellfinder/pull/407 with a torch backend.

**Comment by adamltyson** [comment]
So to summarise, the next release will be pretty much the same accuracy and speed (but easier to maintain for us)?

**Comment by IgorTatarnikov** [comment]
Yup! It might be interesting to include some comparisons using a c-fos brain. I'm curious how the new release will compare to the old release with millions of cell candidates.

**Comment by adamltyson** [comment]
@alessandrofelder do you happen to have any data like this to hand?

I think we should include it if we have it, otherwise we can post something later on. This blog post could end up being longer than the original paper otherwise!

**Comment by adamltyson** [comment]
Due to various updates to other packages, have decided to create a blog post for the new version of the metapackage. This blogpost can be solely about the move to torch.

**Comment by alessandrofelder** [comment]
> @alessandrofelder do you happen to have any data like this to hand?

Yes, I'll DM @IgorTatarnikov the details to it on our internal storage
